integrations-and-supported-tools/optuna/notebooks/Neptune_Optuna_integration.ipynb

```diff
@@ -328,7 +337,10 @@
    "outputs": [],
    "source": [
     "# Create a Neptune run\n",
-    "run = neptune.init(api_token=\"ANONYMOUS\", project=\"common/optuna-integration\")  # you can pass your credentials here\n",
+    "run = neptune.init_run(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,\n",
+    "    project=\"common/optuna-integration\"\n",
+    ")  # you can pass your credentials here\n",
```

Comment:
Not 100% sure about keeping this comment around. We don't want to suggest that they should put their API token here directly, so we'd want to tie it back to the `getpass()` or environment variable strat, but it'd get a bit verbose.

If there are no other ideas, maybe we could just remove it for now.

Reply:
This branch has been merged with #147 

Since we are already informing users how to access and use their token earlier in the notebook, removing this comment